release-notes/8.0/known-issues.md

```diff
@@ -10,4 +10,8 @@ For details about known issues, please refer to the individual repositories:
 
 * [.NET MAUI](https://github.com/dotnet/maui/wiki/Known-Issues/)
 * [Android](https://github.com/xamarin/xamarin-android/wiki/Known-issues-in-.NET)
-* [iOS and macOS](https://github.com/xamarin/xamarin-macios/wiki/Known-issues-in-.NET8)
\ No newline at end of file
+* [iOS and macOS](https://github.com/xamarin/xamarin-macios/wiki/Known-issues-in-.NET8)
```

Comment:
is it in ".NET8" or .NET8 and higher - asking because all links are not specific to version except this one

Reply:
That link is for Xamarin, and was existing before this PR.